src/autocompletion_provider/modelAutocompletionProvider.ts

```diff
@@ -48,16 +48,24 @@ export class ModelAutocompletionProvider
     token: CancellationToken,
     context: CompletionContext
   ): ProviderResult<CompletionItem[] | CompletionList<CompletionItem>> {
-    const linePrefix = document
+    let linePrefix = document
       .lineAt(position)
-      .text.substr(0, position.character);
+      .text.substring(0, position.character);
     if (
       (linePrefix.match(ModelAutocompletionProvider.MODEL_PATTERN) ||
         linePrefix.match(ModelAutocompletionProvider.PACKAGE_PATTERN)) &&
       isEnclosedWithinCodeBlock(document, position)
     ) {
       let quoteFound = false;
       let quote = "";
+      const refString = "ref(";
+      const refStringPosition = linePrefix.indexOf(refString);
```

Comment:
As far as I understand autocomplete is triggered only if `ref` is present in line as coded here
https://github.com/innoverio/vscode-dbt-power-user/blob/5e43c89c8ad81f519ba2315bf8455eceefbbb446/src/autocompletion_provider/modelAutocompletionProvider.ts#L22

Just quotes without `ref` will not trigger autocomplete. 
In case of `ref` without quotes autocomplete works.

I've compiled extension with current fix and using it daily. So far everything works well.